instrumentation/bunny/lib/opentelemetry/instrumentation/bunny/patches/queue.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Bunny
+      module Patches
+        # The Queue module contains the instrumentation patch the Queue#pop method.
+        module Queue
```

Comment:
Is `Queue` typically used directly by clients or is it only used internally? It feels like we're going to end up with nested `process` spans in some cases, but I may be misunderstanding the gem.

Reply:
`.pop` can be used by clients http://rubybunny.info/articles/queues.html#fetching_messages_when_needed_pull_api